schema.ts

```diff
@@ -834,6 +834,37 @@ export interface ObjectSchemaDefinition<K extends string, L extends string> exte
    * scenarios they can be useful.
    */
   includeUnknownProperties?: boolean;
+  /**
+   * The name of a property within {@link properties} that can be used to attribute the source
```

Comment:
"attribute the source" feels pretty vague... maybe "...that will navigate users to more details about this object"?

Reply:
good call